README.md

```diff
@@ -144,7 +144,7 @@ To use this module, you can easily copy the files and put them in your project.
 A more elegant way is to use [git submodule](http://git-scm.com/book/en/Git-Tools-Submodules). Hence, you can keep up with latest improvements. In you plugin directory, do
 
 ```
-git submodule add git://github.com/3nids/qgissettingmanager.git
+git submodule add https://github.com/Septima/qgissettingmanager.git
 ```
 
```

Comment:
I like your changes, but I would keep upstream repo in the readmne. I suppose it was not intended ;)